### PR TITLE
[ENG-1876] Styling login Buttons

### PR DESCRIPTION
**Pull request by yardz**

### CODE REVIEW

**Review by dhruv-kumar-jha — COMMENTED** [review]
@yardz Why is the line padding/margin increased for code? Was there any change in linter/prettier config?

**Comment by yardz** [comment]
> @yardz Why is the line padding/margin increased for code? Was there any change in linter/prettier config?

Probably prettier settings. My IDE applies the current prettier settings every time I save a file. Probably no one run prettier on this project/file.

File: packages/OreIdLoginButton/src/components/LoginProviderButton/LoginProviderButton.tsx [hunk 1]
```diff
@@ -2,75 +2,100 @@ import React, { FunctionComponent, useMemo } from "react";
...
-    };
-  }, []);
+		const providerConfig = useMemo(() => {
+			const metadata = providers.find((Provider) => Provider.id === provider);
+			if (!metadata) {
+				throw Error(
+					`${provider} is not one of the supported providers. Use one of the following: ${providers
+						.map((p) => p.id)
+						.join(", ")}`
+				);
```
Comment (bundance) [inline packages/OreIdLoginButton/src/components/LoginProviderButton/LoginProviderButton.tsx:54]: This definitely isn't right! There's far too much whitespace 🤔

**Review by bundance — CHANGES_REQUESTED** [review]
Something's wrong with your formatting. Each line is using 6 spaces, which is different from the rest of the code, and is adding far too much whitespace everywhere.

File: packages/OreIdLoginButton/src/components/LoginProviderButton/LoginProviderButton.tsx (hunk 1 above)
Comment (yardz) [reply packages/OreIdLoginButton/src/components/LoginProviderButton/LoginProviderButton.tsx]: My prettier was right. The one from the repository that wasn't configured at all.

Just fix in my last 2 commits

File: packages/OreIdLoginButton/package.json [hunk 2]
```diff
@@ -26,7 +27,7 @@
     "react-dom": "^16.8.0 || ^17.0.0|| ^18.0.0"
   },
   "devDependencies": {
-    "rollup-plugin-terser": "^7.0.2",
+    "@aikon/aikon-configs": "^1.0.5",
```
Comment (fractallian) [inline packages/OreIdLoginButton/package.json:30]: This is a public repo. aikon-configs is private and part of our monorepo so I don't think we should add this dependency. This is only here for the prettier config right? We should just copy it. It's really small and doesn't change.

**Review by fractallian — CHANGES_REQUESTED** [review]

File: packages/OreIdLoginButton/package.json (hunk 2 above)
Comment (yardz) [reply packages/OreIdLoginButton/package.json:30]: Aikon-configs should be public. It has only lint and prettier rules...
There is no confidential information of any kind, just things to format the code.

Comment (yardz) [reply packages/OreIdLoginButton/package.json:30]: At least the (published) package should be public

**Review by bundance — APPROVED** [review]
That's a lot of Prettier changes for what I suspect is just a few lines of changed code!

All good though ✅

File: packages/OreIdLoginButton/package.json (hunk 2 above)
Comment (fractallian) [reply packages/OreIdLoginButton/package.json:30]: Ok, I didn't realize it was published as public.

**Review by fractallian — APPROVED** [review]

File: packages/OreIdLoginButton/package.json (hunk 2 above)
Comment (yardz) [reply packages/OreIdLoginButton/package.json:30]: I don't know if it's public. I said it should (and there's no point in not be)

Comment (fractallian) [reply packages/OreIdLoginButton/package.json:30]: I checked, it is published publicly

**Pull request merged**